tests/api/test_core_coord_translation_bh.cpp

```diff
@@ -0,0 +1,419 @@
+                const CoreCoord logical_coords = CoreCoord(x, y, CoreType::TENSIX, CoordSystem::LOGICAL);
+
+                const CoreCoord physical_coords = coordinate_manager->to(logical_coords, CoordSystem::PHYSICAL);
+                const CoreCoord translated_coords = coordinate_manager->to(logical_coords, CoordSystem::TRANSLATED);
+
+                EXPECT_EQ(physical_coords.x, translated_coords.x);
+                EXPECT_EQ(physical_coords.y, translated_coords.y);
+            }
+        }
+    }
+}
+
+// Test that virtual, physical and translated coordinates are the same for all logical PCIE coordinates.
+TEST(CoordinateManager, CoordinateManagerBlackholePCIETranslation) {
+    std::shared_ptr<CoordinateManager> coordinate_manager =
+        CoordinateManager::get_coordinate_manager(tt::ARCH::BLACKHOLE);
+    const tt_xy_pair pcie_grid_size = tt::umd::blackhole::PCIE_GRID_SIZE;
+
+    for (size_t x = 0; x < pcie_grid_size.x; x++) {
+        for (size_t y = 0; y < pcie_grid_size.y; y++) {
+            const CoreCoord arc_logical = CoreCoord(x, y, CoreType::PCIE, CoordSystem::LOGICAL);
+            const CoreCoord arc_virtual = coordinate_manager->to(arc_logical, CoordSystem::VIRTUAL);
+            const CoreCoord arc_physical = coordinate_manager->to(arc_logical, CoordSystem::PHYSICAL);
+
+            EXPECT_EQ(arc_virtual.x, arc_physical.x);
+            EXPECT_EQ(arc_virtual.y, arc_physical.y);
+        }
+    }
+}
+
+// Test that virtual, physical and translated coordinates are the same for all logical ARC coordinates.
+TEST(CoordinateManager, CoordinateManagerBlackholeARCTranslation) {
+    std::shared_ptr<CoordinateManager> coordinate_manager =
+        CoordinateManager::get_coordinate_manager(tt::ARCH::BLACKHOLE);
+    const tt_xy_pair arc_grid_size = tt::umd::blackhole::ARC_GRID_SIZE;
+
+    for (size_t x = 0; x < arc_grid_size.x; x++) {
+        for (size_t y = 0; y < arc_grid_size.y; y++) {
+            const CoreCoord arc_logical = CoreCoord(x, y, CoreType::ARC, CoordSystem::LOGICAL);
+            const CoreCoord arc_virtual = coordinate_manager->to(arc_logical, CoordSystem::VIRTUAL);
+            const CoreCoord arc_physical = coordinate_manager->to(arc_logical, CoordSystem::PHYSICAL);
+            const CoreCoord arc_translated = coordinate_manager->to(arc_logical, CoordSystem::TRANSLATED);
+
+            EXPECT_EQ(arc_virtual.x, arc_physical.x);
+            EXPECT_EQ(arc_virtual.y, arc_physical.y);
+
+            EXPECT_EQ(arc_virtual.x, arc_translated.x);
+            EXPECT_EQ(arc_virtual.y, arc_translated.y);
+        }
+    }
+}
+
+// Test ethernet coordinate translation.
+TEST(CoordinateManager, CoordinateManagerBlackholeETHTranslation) {
+    std::shared_ptr<CoordinateManager> coordinate_manager =
+        CoordinateManager::get_coordinate_manager(tt::ARCH::BLACKHOLE);
+    const tt_xy_pair eth_grid_size = tt::umd::blackhole::ETH_GRID_SIZE;
+
+    const size_t eth_translated_coordinate_start_x = 20;
```

Comment:
Can we define these constants 20 and 25 somewhere in the coord_manager or soc descriptor?

Reply:
same as above